packages/app/cypress/e2e/studio/helper.ts

```diff
@@ -1,3 +1,17 @@
+export function loadExperimentalStudio ({ specName = 'spec.cy.js', cliArgs = [''], enableCloudStudio = false, specSelector = 'data-cy-row' } = {}) {
```

Comment:
Or maybe something like:
```suggestion
export function loadAndRunSpec ({ specName = 'spec.cy.js', cliArgs = [''], enableCloudStudio = false, specSelector = 'data-cy-row' } = {}) {
```